tests/installation/partitioning_expert.pm

```diff
@@ -17,29 +17,30 @@ use base "y2logsstep";
 use testapi;
 
 sub run() {
-    send_key $cmd{expertpartitioner};
+    wait_screen_change { send_key 'alt-x-c' }; # Expert partitioner, using current proposal
```

Comment:
Since Tumbleweed is the default version, if the hotkey changed, the constant _expertpartitioner_ should be changed and the test modules that use that constant should be updated.
@okurz, am I right?

```bash
copland:~/github/os-autoinst-distri-opensuse$ git grep expertpartitioner
lib/partition_setup.pm:    send_key match_has_tag('bsc#1054478') ? 'alt-x' : $cmd{expertpartitioner};
lib/susedistribution.pm:      expertpartitioner alt-e
tests/installation/partitioning.pm:            $cmd{expertpartitioner} = 'alt-x';
tests/installation/partitioning_expert.pm:    send_key $cmd{expertpartitioner};
tests/installation/partitioning_raid.pm:    send_key(is_storage_ng() ? $cmd{expertpartitioner} : $cmd{createpartsetup});
tests/installation/partitioning_resize_root.pm:    send_key $cmd{expertpartitioner};
```

@rwx788, could this be a first example to use element IDs? Is the feature ready to use it here?

Reply:
@SergioAtSUSE yes, you are right :)

Reply:
@SergioAtSUSE PRs were merged, not sure when exactly this pop-up feature will reach distribution though.